src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.md

```diff
@@ -3,50 +3,50 @@ Sr. No. | Rule ID | Title | Category | Enabled | CodeFix | Description |
 --------|---------|-------|----------|---------|---------|--------------------------------------------------------------------------------------------------------------|
 1 | [CA1000](https://docs.microsoft.com/visualstudio/code-quality/ca1000-do-not-declare-static-members-on-generic-types) | Do not declare static members on generic types | Design | True | False | When a static member of a generic type is called, the type argument must be specified for the type. When a generic instance member that does not support inference is called, the type argument must be specified for the member. In these two cases, the syntax for specifying the type argument is different and easily confused. |
 2 | [CA1001](https://docs.microsoft.com/visualstudio/code-quality/ca1001-types-that-own-disposable-fields-should-be-disposable) | Types that own disposable fields should be disposable | Design | True | True | A class declares and implements an instance field that is a System.IDisposable type, and the class does not implement IDisposable. A class that declares an IDisposable field indirectly owns an unmanaged resource and should implement the IDisposable interface. |
-3 | [CA1003](https://docs.microsoft.com/visualstudio/code-quality/ca1003-use-generic-event-handler-instances) | Use generic event handler instances | Design | False | True | A type contains an event that declares an EventHandler delegate that returns void, whose signature contains two parameters (the first an object and the second a type that is assignable to EventArgs), and the containing assembly targets Microsoft .NET Framework?2.0. |
+3 | [CA1003](https://docs.microsoft.com/visualstudio/code-quality/ca1003-use-generic-event-handler-instances) | Use generic event handler instances | Design | False | False | A type contains an event that declares an EventHandler delegate that returns void, whose signature contains two parameters (the first an object and the second a type that is assignable to EventArgs), and the containing assembly targets Microsoft .NET Framework?2.0. |
```

Comment:
:bulb: This would be easier to glance over if False was changed to :x: and True was changed to ✔

Reply:
These are auto-generated/updated files, we don't expect reviewing changes to these files.